### ISSUE TITLE
VGG 16 LR?

### ISSUE THREAD

**Issue by a-maci** [ppwwyyxx/tensorpack#299]
Any chance you have trained VGG-16 (or -19) and know the learning rate schedule?
I cant reproduce the results mentioned in the paper with the learning rates they mentioned.

**Comment by ppwwyyxx** [comment]
I don't have any. I do know that VGGs are very hard to reproduce.

**Comment by a-maci** [comment]
Thanks.
